.github/workflows/documentation.yml

```diff
@@ -27,8 +27,12 @@ jobs:
         run: "rm composer.json"
 
       - name: "Require phpdocumentor/guides-cli"
-        # We use the same version constraint as in doctrine/doctrine-website
-        run: "composer require --dev phpdocumentor/guides-cli '^1.4' --no-update"
+        # We use the same versions as in doctrine/doctrine-website
+        run: >
+          composer require --dev
+          phpdocumentor/guides-cli:1.7.0
+          phpdocumentor/filesystem:1.7.0
+          --no-update
```

Comment:
Personally, I find this YAML syntax quite misleading. It reads like multiple commands, and one cannot just copy-paste it and run in their shell. Could we keep it as a single line? It's not that long.

Reply:
I will try to remember to do that next try I upgrade these versions.